### PR TITLE
GUI - Allow InputText text outline

### PR DESCRIPTION
**Pull request by RaananW**
See https://forum.babylonjs.com/t/inputtextarea-onfocusselectall-does-not-work/48351

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14819/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14819/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14819/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14819/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14819/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14819/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14819/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14819/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14819/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14819/merge/testResults/webgl2playwright/index.html

File: packages/dev/gui/src/2D/controls/inputText.ts [hunk 1]
```diff
@@ -65,6 +67,42 @@ export class InputText extends Control implements IFocusableControl {
...
+    /**
+     * Gets or sets outlineWidth of the text to display
+     */
+    public get outlineWidth(): number {
+        return this._outlineWidth;
+    }
+
+    /**
+     * Gets or sets outlineWidth of the text to display
+     */
```
Comment (Popov72) [inline packages/dev/gui/src/2D/controls/inputText.ts:79]: I think you don't have to repeat the doc for a get/set pair(?)

File: packages/dev/gui/src/2D/controls/inputText.ts [hunk 2]
```diff
@@ -65,6 +67,42 @@ export class InputText extends Control implements IFocusableControl {
...
+    /**
+     * Gets or sets outlineColor of the text to display
+     */
+    public get outlineColor(): string {
+        return this._outlineColor;
+    }
+
+    /**
+     * Gets or sets outlineColor of the text to display
+     */
```
Comment (Popov72) [inline packages/dev/gui/src/2D/controls/inputText.ts:97]: Same.

File: packages/dev/gui/src/2D/controls/inputText.ts (hunk 1 above)
Comment (RaananW) [reply packages/dev/gui/src/2D/controls/inputText.ts:79]: Was just a copy and paste from the other one. And you are right, it's not really needed

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14819/merge/testResults/webgpuplaywright/index.html

**Pull request merged**